openstack/identity/v3/registered_limits/testing/fixtures.go

```diff
@@ -0,0 +1,219 @@
+package testing
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/gophercloud/gophercloud/openstack/identity/v3/registered_limits"
+	th "github.com/gophercloud/gophercloud/testhelper"
+	"github.com/gophercloud/gophercloud/testhelper/client"
+)
+
+// ListOutput provides a single page of List results.
+const ListOutput = `
+{
+    "links": {
+        "self": "http://10.3.150.25/identity/v3/registered_limits",
+        "previous": null,
+        "next": null
+    },
+    "registered_limits": [
+        {
+            "resource_name": "volume",
+            "region_id": RegionOne,
```

Comment:
The `"invalid character 'R' looking for beginning of value"` you [see in the unit tests](https://github.com/gophercloud/gophercloud/actions/runs/4978055778/jobs/8907869530?pr=2616#step:6:138) comes from missing quotes around `RegionOne`.